### ISSUE TITLE
model.summary() cannot be displayed normally

### ISSUE THREAD

**Issue by Huffman-cotdom** [microsoft/vscode-jupyter#4336]
# 定义一个Sequential模型，包含3层
model = keras.Sequential(
    [
        # 第一层：激活函数为relu,权重初始化为he_normal
        layers.Dense(3, activation="relu",
                     kernel_initializer="he_normal", name="layer1",input_shape=(3,)),
        # 第二层：激活函数为relu,权重初始化为he_normal
        layers.Dense(2, activation="relu",
                     kernel_initializer="he_normal", name="layer2"),
        # 第三层（输出层）：激活函数为sigmoid,权重初始化为he_normal
        layers.Dense(2, activation="sigmoid",
                     kernel_initializer="he_normal", name="layer3"),
    ],
    name="my_Sequential"
)
model.summary()
+  model.summary() cannot be displayed normally

**Comment by claudiaregio** [comment]
Hi @Huffman-cotdom, 
Any chance this issue could be translated to English? In the meantime I will try to find someone who can identify this language. 

Could you also provide more details or screenshots regarding the issue? To help troubleshoot could you provide a screenshot of what you are seeing vs. what should be seen?
